src/main/java/MongoDB.java

```diff
@@ -0,0 +1,59 @@
+import com.mongodb.*;
```

Comment:
why do you need a wild card import?Do you really need all the classes from the com.mongodb package?